assets/src/edit-story/components/richText/karma/_utils.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app/story';
+
+export function initHelpers(data) {
+  async function getElements() {
+    const storyContext = await data.fixture.renderHook(() => useStory());
+    return storyContext.state.currentPage.elements;
+  }
+
+  async function addInitialText() {
+    await data.fixture.events.click(data.fixture.editor.library.textAdd);
+
+    const [{ id: bgId }, { id: textId }] = await getElements();
+    data.bgId = bgId;
+    data.textId = textId;
+
+    await data.fixture.editor.canvas.framesLayer
+      .frame(data.textId)
+      .waitFocusedWithin();
+  }
+
+  function getTextContent() {
+    return data.fixture.editor.canvas.framesLayer.frame(data.textId)
+      .textContent;
+  }
+
+  async function richTextHasFocus() {
+    await data.fixture.editor.canvas.waitFocusedWithin();
+  }
+
+  function repeatPress(key, count) {
+    let remaining = count;
+    const press = () => {
+      if (remaining === 0) {
+        return true;
```

Comment:
Nit: `Promise.resolve(true)` to have a consistent return type?

Reply:
Fixed in 59e09b67d.